src/main/java/edu/asu/plp/tool/backend/isa/events/MemWatchResponseEvent.java

```diff
@@ -2,26 +2,47 @@
 
 import javafx.beans.property.LongProperty;
 
+/**
+ * This class defines the response object for Memory watch event
+ */
 public class MemWatchResponseEvent {
 	
 	private boolean success;
 	private long watchedAddress;
 	private LongProperty memObject;
 
+	/**
+	 * Constructor for Memory watch response object
+	 * @param success
+	 * @param watchedAddress
+	 * @param memObject
+	 */
 	public MemWatchResponseEvent(boolean success, long watchedAddress, LongProperty memObject) {
 		this.success = success;
 		this.watchedAddress = watchedAddress;
 		this.memObject = memObject;
 	}
 
+	/**
+	 * getter for success flag
+	 * @return
+	 */
 	public boolean isSuccess() {
 		return success;
 	}
-	
+
+	/**
+	 * getter for address that is watched
+	 * @return
```

Comment:
Indicate the return value, like `@return a valid address when the watch request succeed, otherwise the address is invalid`